src/top-bar/context-select/context-select.test.js

```diff
@@ -74,7 +80,13 @@ describe('<ContextSelect>', () => {
         })
 
         it('calls onClose when the backdrop layer is clicked', () => {
-            wrapper.find(Popover).dive().simulate('click')
+            wrapper
+                .find(Popover)
+                .dive()
+                .find(Layer)
+                .dive()
+                .find('.backdrop')
```

Comment:
I'm failing to understand how this is related to ARM64 support. I suppose that you encountered an issue with this test and these changes fixed them. I still have no idea why the code would pass on non-ARM64 machines but fail on your computer without your additions

Reply:
See https://github.com/dhis2/approval-app/pull/231#discussion_r1101306757